### PR TITLE
Ship the .aar with the NuGet for android

### LINKED ISSUE #2444: [QUESTION] .Net 8 is supported?
I am currently testing .net 8 maui project. That project have the SkiaSharp dependency. I can't run the project.

I am getting the following error

System.BadImageFormatException: 'Invalid field token 0x040000a1' 

image

### PR DESCRIPTION
**Pull request by dellis1972**
**Description of Change**

**Bugs Fixed**

- Fixes #2444

**API Changes**

None.

**Behavioral Changes**

None.

**Required skia PR**

None.

**PR Checklist**

- [ ] Has tests (if omitted, state reason in description)
- [x] Rebased on top of main at time of PR
- [x] Merged related skia PRs
- [x] Changes adhere to coding standard
- [x] Updated documentation

### CODE REVIEW

**Comment by jonathanpeppers** [comment]
@mattleibow can we get this in for .NET 8 customers?

File: nuget/SkiaSharp.Views.nuspec [hunk 1]
```diff
@@ -106,6 +106,7 @@ Please visit https://go.microsoft.com/fwlink/?linkid=868517 to view the release
     <file src="lib/net6.0-android/SkiaSharp.Views.Android.dll" target="lib/net6.0-android30.0/SkiaSharp.Views.Android.dll" />
     <file src="lib/net6.0-android/SkiaSharp.Views.Android.pdb" target="lib/net6.0-android30.0/SkiaSharp.Views.Android.pdb" />
     <file src="lib/net6.0-android/SkiaSharp.Views.Android.xml" target="lib/net6.0-android30.0/SkiaSharp.Views.Android.xml" />
+    <file src="lib/net6.0-android/SkiaSharp.Views.Android.aar" target="lib/net6.0-android30.0/SkiaSharp.Views.Android.aar" />
```
Comment (mattleibow) [inline nuget/SkiaSharp.Views.nuspec:109]: This packs the file in the NuGet package.

File: source/SkiaSharp.Build.targets [hunk 2]
```diff
@@ -187,11 +187,13 @@ internal partial class VersionConstants {
     <ItemGroup Condition=" '$(PackagingLocation)' == '' ">
       <_CopyItems Include="$(TargetPath)" Dest="nuget\lib\$(PackagingPlatform)\$(TargetFileName)" />
       <_CopyItems Include="$(TargetDir)$(TargetName).xml" Dest="nuget\lib\$(PackagingPlatform)\$(TargetName).xml" Condition=" Exists('$(TargetDir)$(TargetName).xml') " />
+      <_CopyItems Include="$(TargetDir)$(TargetName).aar" Dest="nuget\lib\$(PackagingPlatform)\$(TargetName).aar" Condition=" Exists('$(TargetDir)$(TargetName).aar') " />
```
Comment (mattleibow) [inline source/SkiaSharp.Build.targets:190]: This copies from the bin directory into the artifacts directory.

**Review by mattleibow — APPROVED** [review]

**Review by mattleibow — APPROVED** [review]

**Pull request merged**